geode-core/src/main/java/org/apache/geode/redis/internal/executor/sortedset/ZLexCountExecutor.java

```diff
@@ -103,21 +103,23 @@ public void executeCommand(Command command, ExecutionHandlerContext context) {
   private int getCount(ByteArrayWrapper key, Region<ByteArrayWrapper, DoubleWrapper> keyRegion,
       ExecutionHandlerContext context, ByteArrayWrapper start, ByteArrayWrapper stop,
       boolean startInclusive, boolean stopInclusive) throws Exception {
-    if (start.equals("-") && stop.equals("+"))
+    final ByteArrayWrapper minus = new ByteArrayWrapper(Coder.stringToBytes("-"));
```

Comment:
Maybe even hoist these out of the method to be private static final constants for the class to use. Or if this class and the others have a shared super class then you could pull them all the way up to the super class.